### DISCUSSION TITLE
Connecting to zigbee2mqtt running on a separate device

### DISCUSSION THREAD

**Discussion by JRJSmith** [Koenkk/zigbee2mqtt#11344] [General]
I apologise for a 'noob' question but I haven't been able to find the answer despite trying numerous searches here and elsewhere.

I am running Home Assistant as a Virtual Machine on my Synology NAS. I was using a Conbee II with both the Mosquitto broker and Zigbee2mqtt Add-ons but, whenever the Synology NAS restarted, the Home Assistant VM didn't pick up the Conbee II without me having to edit the VM's >> Others to mount the USB device. I could find no way to make the connection persistent over reboots.

I decided to set up a Raspberry Pi as the Zigbee Coordinator instead, intending to make it Read-Only once everything is working properly. 

The Raspberry Pi Zigbee Coordinator connects with the Home Assistant VM's Mosquitto broker without issue. However, the zigbee2mqtt instance on the Home Assistant VM fails to start because there is no Zigbee USB device connected. 

I liked the capability of zigbee2mqtt to draw a map of the Zigbee mesh network when the Home Assistant VM hosted the Zigbee Coordinator and I would like to achieve the same capability using the RPi Zigbee Coordinator but cannot see how to achieve it if I cannot start zigbee2mqtt on the Home Assistant VM.

I've seen a separate post where Koenkk said that it wasn't possible to start zigbee2mqtt without a zigbee adapter but later in the post, he was beginning to acknowledge some use cases.

My question is can what I am seeking be achieved or am I missing something fundamental due to my inexperience or lack of understanding? Is there an alternative that I have missed?

Many thanks in advance for any assistance that may be offered. 

Best regards
Joe

**Comment by aldur11** [comment]
So you running HA on a VM on your NAS and you want to run z2m on a raspberry pi, right? If that's the case you don't run z2m on HA, you just setup a mosquitto broker on HA and point the z2m instance on your raspberry pi at the mosquitto broker.

**Reply by JRJSmith** [reply]
Many thanks for your response and the conclusion that I do not run z2m on HA is not unexpected. 

However, this raises a connected question: If z2m is running on the RPi but not on the HA VM, is there any way that I can display a map of the Zigbee mesh on HA, please?

Best regards and thanks again
Joe

**Reply by aldur11** [reply]
I don't think there's a way to view the map in HA when using an external z2m instance. However, you can just view the map in the z2m front end.

**Reply by JRJSmith** [reply]
Sorry for the delayed reply and thanks for your response.
It illustrates how inexperienced I am with this that I hadn't even considered the z2m front end. I'll investigate that further.
Your preparedness to assist is very much appreciated.

Best regards and thanks again
Joe

**Reply by aldur11** [reply]
No worries, and I'm glad to hear that it worked out for you :)
